demo/api.min.js

```diff
@@ -37,6 +35,45 @@ function monthNamesExample() {
   monthNamesExample.monthNames = spanishMonths;
 }
 
+function populateSlotContentExample() {
```

Comment:
This function should have inline comments.

Reply:
Added.